great_expectations/render/renderer/v3/suite_edit_notebook_renderer.py

```diff
@@ -372,11 +372,7 @@ def render(
 
         suite_name: str = suite.expectation_suite_name
 
-        if (
-            batch_request
-            and isinstance(batch_request, dict)
-            and BatchRequest(**batch_request)
-        ):
```

Comment:
@NathanFarmer This check was making sure that the `BatchRequest` is not malformed.  Are you confident that we can remove it?  Thanks.

Reply:
@alexsherstinsky this is an important requirement for this PR. We want to accept malformed `BatchRequest`s that are passed by `great_expectations.yml` when working with the CLI workflow. This will enable to user to fix their `BatchRequest` in the `suite edit` notebook instead of seeing an error raised in CLI.

Reply:
@NathanFarmer I see -- thanks for clarifying.

Reply:
@alexsherstinsky it is both YML and CLI params. `datasource`s, `data_connector`s, and `assets` are defined in YML. They are selected by CLI prompts, but in the event of an incorrect configuration errors are thrown and any decisions made by the user in previous CLI prompts are lost.